### PR TITLE
[CBRD-20418] Disable log compression for log records with size less than 255 bytes

### PR DESCRIPTION
**Pull request by razvanradulescu**

### CODE REVIEW

**Comment by cubridci** [comment]
Build comment file: 
--------------
#### File: src/base/system_parameter.c (1 incorrect indentation(s) found)
```
  2018 static int prm_log_zip_min_size_compress_upper = 16 * 1024 * 1024;	/* 16 M */
```
--------------
**Please run 'indent -l120 -lc120' to correct the indentation**

--------------

**Comment by eseokoh** [comment]
I'd like to propose to remove the parameter. When DBA changes it, database may not be recovered. I don't also think DBA can find a proper value. I think it is better to use a static const.

**Comment by eseokoh** [comment]
My argument on recovery might be wrong. 
However, I still prefer to static const.

File: src/transaction/log_page_buffer.c [hunk 1]
```diff
@@ -314,9 +314,7 @@ static int log_data_length = 0;
 
 LOG_LSA NULL_LSA = { NULL_PAGEID, NULL_OFFSET };
 
-#define LOG_ZIP_MIN_SIZE_TO_COMPRESS log_zip_min_size_to_compress
-
-static int log_zip_min_size_to_compress = 0;
+static int log_zip_min_size_to_compress = 255;
```
Comment (eseokoh) [inline src/transaction/log_page_buffer.c]: @razvanradulescu 

Would you please rename it as `log_Zip_min_size_to_compress` to follow naming convention for globals?

**Comment by eseokoh** [comment]
@razvanradulescu 

Thanks for updates. :)
Would you merge it?

**Pull request merged**